.github/workflows/release.yml

```diff
@@ -205,7 +205,7 @@ jobs:
           # TICKET_TYPE: ${{ env.ticket_type }}
```

Comment:
The code changes include modifying the PR_DESCRIPTION assignment statement to correct the syntax by removing the double curly braces around PR_DESCRIPTION. This fix ensures that the variable is properly referenced when being used later in the script. The change also involves updating the command to exclude the first and last lines of the PR_DESCRIPTION using sed. Overall, the changes look good and address the issues effectively.